tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py

```diff
@@ -0,0 +1,491 @@
+# -*- coding: utf8 -*-
+"""
+Test cases here performs Prometheus queries directly without a workload, to
+check that OCS Monitoring is configured and available as expected.
+"""
+
+import logging
+import time
+
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.ocs import constants, defaults, ocp
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.prometheus import PrometheusAPI
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.post_deployment
+@pytest.mark.polarion_id("OCS-1261")
+def test_monitoring_enabled():
+    """
+    OCS Monitoring is enabled after OCS installation (which is why this test
+    has a post deployment marker) by asking for values of one ceph and one
+    noobaa related metrics.
+    """
+    prometheus = PrometheusAPI()
+
+    # ask for values of ceph_pool_stored metric
+    logger.info("Checking that ceph data are provided in OCS monitoring")
+    result = prometheus.query('ceph_pool_stored')
+    msg = "check that we actually received some values for a ceph query"
+    assert len(result) > 0, msg
+    for metric in result:
+        _, value = metric['value']
+        assert int(value) >= 0, "number bytes in a pool isn't negative integer"
+    # additional check that values makes at least some sense
+    logger.info(
+        "Checking that size of ceph_pool_stored result matches number of pools")
+    ct_pod = pod.get_ceph_tools_pod()
+    ceph_pools = ct_pod.exec_ceph_cmd("ceph osd pool ls")
+    assert len(result) == len(ceph_pools)
+
+    # TODO: remove BZ 1790558 workaround (noobaa is not immediatelly ready just
+    # after installation)
+    hack_sleep = 600  # in seconds
+    logger.info(
+        f"BZ 1790558 workaround: going to sleep for {hack_sleep} seconds")
+    time.sleep(hack_sleep)
```

Comment:
Would it be possible to make the sleep more smarter? (Check the availability of nooba periodically and fail after 10 minutes? It might significantly shorten duration of the test.)

Reply:
I could try to optimize it, but only later based on evaluation of BZ 1790558.